### ISSUE TITLE
New problem: Dynamic Tanh (DyT)

### ISSUE THREAD

**Issue by turkunov** [Open-Deep-ML/DML-OpenProblem#386]
Normalization layers are ubiquitous in modern neural networks and have long been considered essential. A fresh paper (https://arxiv.org/pdf/2503.10622) demonstrates that layer normalization in Transformers often produces tanh-like, S-shaped input-output mappings. By incorporating a new layer replacement for normalization called "Dynamic Tanh" (DyT for short), Transformers without normalization can match or exceed the performance of their normalized counterparts, mostly without hyperparameter tuning.

I can take a closer look and cook up a solution in a couple of days.

**Comment by moe18** [comment]
that would make a great question!
